compile.go

```diff
@@ -89,6 +89,7 @@ var (
 		"math",
 		"strings",
 		"regexp",
+		"time",
```

Comment:
Lets not add the `time` package to the prelude. People can import it if they need it. See https://github.com/influxdata/flux/issues/1480

Reply:
Fixing #1480 is blocked on #1413, as such lets not block this work. We can add `time` to the prelude for now as doing so does not make fixing #1480 any harder.